packages/opentelemetry-plugin-postgres/src/pg.ts

```diff
@@ -112,52 +114,72 @@ export class PostgresPlugin extends BasePlugin<typeof pgTypes> {
 
         // Bind promise to parent span and end the span
         if (result instanceof Promise) {
-          return plugin._tracer.bind(result
-          .then((result: unknown) => {
-            // Return a pass-along promise which ends the span and then goes to user's orig resolvers
-            return new Promise((resolve, _) => {
-              span.setStatus({ code: CanonicalCode.OK });
-              span.end();
-              resolve(result);
-            });
-          })
-          .catch((error: Error) => {
-            return new Promise((_, reject) => {
-              span.setStatus({ code: CanonicalCode.UNKNOWN })
-              span.end();
-              reject(error);
-            });
-          }));
+          return plugin._tracer.bind(
```

Comment:
FYI: the async hooks scope manager doesn't handle binding a scope to a promise, so if it's working as-is, we could remove the `.bind` call.

Reply:
Good point. It works without it so I removed these binds